src/snowflake/connector/network.py

```diff
@@ -451,6 +451,7 @@ def request(
         method: str = "post",
         client: str = "sfsql",
         timeout: int | None = None,
+        params: dict | None = None,
```

Comment:
I think this is technically a BCR since network.request is a public API and we introduced a new positional parameter in the middle of existing params.

what about making it a key word argument?